### PR TITLE
Added dont ask again vote to reject

### PR DESCRIPTION
**Pull request by Tguntenaar**
This PR adds a test 
- to reject an payment request with don't ask me again enabled. 
- moves the voting tests to approve to a seperate file.
- fixed the missspelling of COLUMNS_VISIBILITY
- removes unneccesary awaits in tests

# Approve
[video-1.webm](https://github.com/user-attachments/assets/d144f8c0-10ff-4760-a78d-e2cbae8cbec8)

# Reject
[video-2.webm](https://github.com/user-attachments/assets/16ecf337-fba2-4407-93e7-888f2aa09d13)

TODO: next PR

### CODE REVIEW

**Comment by Tguntenaar** [comment]
@race-of-sloths include

**Comment by race-of-sloths** [comment]
### ⏸️ This repository is on pause
@Tguntenaar please reach out to the maintainer to unpause this repo with `@race-of-sloths unpause` command

**Review by Megha-Dev-19 — APPROVED** [review]
LGTM, left a comment

**Comment by Megha-Dev-19** [comment]
@race-of-sloths unpause

File: playwright-tests/tests/payments/vote-on-request.spec.js [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+    await transaction_toast.waitFor({ state: "detached", timeout: 10000 });
+    await expect(transaction_toast).not.toBeVisible();
+    await page
+      .locator("li")
+      .filter({ hasText: "History" })
+      .locator("div")
+      .click();
+    await expect(await page.getByText("Funded").first()).toBeVisible({
+      timeout: 10_000,
+    });
```
[the remark this answers is not in the document]
Comment (Tguntenaar) [reply playwright-tests/tests/payments/vote-on-request.spec.js:133]: Yea that makes sense will make those changes.

[the remark this answers is not in the document]
Comment (Tguntenaar) [reply playwright-tests/tests/payments/vote-on-request.spec.js]: I'm testing locally and on `treasury-testing-infinex.near` but don't see any highlighting. I'm using mynearwallet, did you test it with meteor wallet?

**Review by Tguntenaar — COMMENTED** [review]
@petersalomonsen could you review?

**Comment by petersalomonsen** [comment]
@race-of-sloths score 5

**Review by petersalomonsen — APPROVED** [review]

**Pull request merged**